### PR TITLE
feat(api): Expose event used for sending browser support

### PR DESCRIPTION
**Pull request by vp8x8**

### CODE REVIEW

File: modules/API/API.js [hunk 1]
```diff
@@ -1355,6 +1357,19 @@ class API {
...
 
+    /**
+     * Notify external application (if API is enabled) wether the used browser is supported or not.
+     *
+     * @param {boolean} supported - If browser is supported or not.
+     * @returns {void}
+     */
+    notifyBrowserSupport(supported: boolean) {
+        this._sendEvent({
+            name: 'browser-support',
```
Comment (saghul) [inline modules/API/API.js:1368]: Don't we want a constant defined in external_api.js and a method to check it?

Comment (vp8x8) [reply modules/API/API.js:1368]: Yeah, I guess that's a better way. It did not cross my mind 😊

Comment (vp8x8) [reply modules/API/API.js:1368]: @saghul Tried to add a method but although it looks like we have tree shaking the bundle for external_api increases 10 fold to 300k. I don't this is something we want...

Comment (saghul) [reply modules/API/API.js:1368]: Sure

**Review by saghul — APPROVED** [review]

**Pull request merged**